vagrant-pxe-harvester/Vagrantfile

```diff
@@ -80,4 +79,29 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       end
     end
   end
+
+  # Rancher
+  # https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-requirements#docker
+  config.vm.define :rancher do |rancher|
+    rancher.vm.box = 'generic/debian10'
+    rancher.vm.hostname = 'rancher'
+    rancher.vm.network 'private_network',
+      ip: @settings['rancher_config']['ip'],
+      libvirt__network_name: 'harvester',
+      libvirt__dhcp_enabled: false
+
+      rancher.vm.provider :libvirt do |libvirt|
+      libvirt.cpu_mode = 'host-passthrough'
+      libvirt.memory = '4096'
+      libvirt.cpus = '1'
```

Comment:
Just a suggestion is perhaps we can provide the ability for user to specify the cpu and memory of the Rancher instance VM in the settings.yaml to provide more scalability

Reply:
Fixed in commit [4a8a903](https://github.com/harvester/ipxe-examples/pull/66/commits/4a8a9034e343338c0f01e7561abb05026d6e5dcd).